_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,144 @@
+
+![Image: Shrink-configure]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-configure.jpg){:.img-fluid }
+
+All of the settings specified above will also be shown in the JSON editor under the **Advanced settings** section. You can also specify additional index settings for the new shrunken index in the JSON editor. For example, you can set both `index.routing.allocation.require._name` and `index.blocks.write` to `null` as seen in the following image. This will clear the allocation requirement and the index write block, which are copied from the source index.
+
+
+![Image: Shrink-advanced-settings]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-advanced-settings.jpg){:.img-fluid }
+
+Once you have specified all of the settings for the new shrunken index, choose the **Shrink** button to trigger the shrink operation. You will see a notification indicating that the shrink operation has started successfully as seen in the following image.
+
+![Image: Shrink-started]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-started.jpg){:.img-fluid }
+
+After the shrink operation is completed, which may take anywhere from minutes to hours, a new notification will be displayed stating that the source index has been successfully shrunken as seen in the following image.
+
+![Image: Shrink-completed]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-completed.jpg){:.img-fluid }
+
+### Open and close index operation
+
+You can select multiple indexes, except for the backing indexes of a data stream, to open or close.
+
+If you no longer need to read or search old indexes but do not want to delete them, you can use the **Close** operation to close indexes. This will maintain the data, but will have a small overhead on the cluster. Additionally, when you want to add a new analyzer to an existing index, you must close the index, define the analyzer, and then open the index. A closed index is blocked for read and write operations, so you must type the word "close" to confirm your action as seen in the following image.
+
+![Image: Close-index]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/close-index.jpg){:.img-fluid }
+
+Opening an index is easier as you can select multiple indexes to open even if some indexes are already open as seen in the following image.
+
+![Image: Open-index]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/open-index.jpg){:.img-fluid }
+
+### Split index
+
+When you choose to split an index, the status of that index is checked. If the index is not able to be split, actions that the user can take to make the index ready for splitting are provided. A list of shard numbers that the index can be split into is also provided so that the user does not need to calculate it manually.
```

Comment:
```suggestion
When you choose to split an index, the status of that index is checked. If the index is not able to be split, actions that you can take to prepare the index for splitting are provided. A list of shard numbers that the index can be split into is also provided so that you do not need to calculate it manually.
```

Reply:
Last sentence: Instead of "it", let's name what we're referring to. "the numbers"?

Reply:
Yes, "the shard numbers" .